editor/src/clj/editor/outline_view.clj

```diff
@@ -489,7 +490,8 @@
                             (ui/add-style! "visibility-toggle")
                             (.addEventFilter MouseEvent/MOUSE_PRESSED ui/ignore-event-filter)
                             (AnchorPane/setRightAnchor 0.0))
-        text-label (Label.)
+        text-label (doto (Label.)
+                     (ui/bind-double-click! :open))
```

Comment:
Moved the double click handler on the label to avoid opening the resource when we double click on the visibility toggle. I think this move makes sense regardless of the bug.